CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+Rules. You gain 3 Contagions. 
+
+    Their Eye: The target loses one of their eyes. They take a permanent -2 to
+their Perception and receive 60 damage. Their to hit score is reduced by 2. 
+You gain 1 Contagion.
+
+    Their Spleen: The target loses their spleen. They take 60 damage and their
+Fortitude is permanently reduced by 2. You gain 2 Contagions.
+
+    Their Lymph Node: The target loses their lymph node. They lose 40 health and
+120 Nanites. You gain 2 Contagions.
+
+==============================
+Hippocratic Tree
+==============================
+
+==Patch Em Up ==
+Adrenaline Reward: 40 Points
+Contagion Reward: 1 Contagion 
+Range:			Touch
+Action:			1 Action
+    One of the target's wounds are healed. This trick only works on biologic 
+patients.
+
+==Defib ==
+Adrenaline Reward:	80 Points
+Contagion Reward: 2 Contagion
+Range: Touch
+Action: 1 Action
+    Only usable on an person that is bleeding out. The person is no longer 
+downed and has 30 hp, rather than having to stabilize the target and then heal 
+them as a separate action. This trick only works on biologic patients.
+
+==Rub Some Dirt In It ==
+Adrenaline Reward:	30*n Points
+Contagion Reward: n (max 3) Contagion
+Range: Touch
+Action: 1 Action
+    The patient is healed by the Doc's heal roll / n, where n is the number of 
+wounds that the patient has. This trick only works on biologic patients.
+
+==Remote Surgery ==
+Adrenaline Reward:	20
+Range:			20m
+Action:			2 Actions
+    One of the patient's wounds are healed for half of the Doc's heal roll. This 
+trick only works on biologic patients.
+
+==Remote Defib ==
+Adrenaline Reward:	65 Points
+Range:			25m
+Action:			2 Actions
+    The patient is no longer bleeding out and has 20 health. This trick only 
+works on biologic patients.
+
+==Cortical Stimulator ==
+Adrenaline Reward:	200 Points
+Contagion Reward: 4 Contagions
+Range: Touch
+Action: 4 Actions
+    Can only be used on a biologic patient who's head is intact who has bled out
+ and died less than 8 turns ago. The Doctor makes a DC 60 heal check. If 
+successful, the patient is now bleeding out. If the patient is saved, gets the 
+feat 'Back from the Dead'. Characters with the 'Back From the Dead' feat have a
+permanent -1m per action move speed, -10 maximum health and can never again be 
+revived by Cortical Stimulator.
+
+==P.E.G.s ==
+Adrenaline Reward:	40 Points
+Contagion Reward: 1 Contagion
+Range:			Touch
+Action:			4 Actions
+    Stands for Persistent Emergent Generalists, a type of surgical nanobot that 
+stays in the patient's bloodstream and continues to administer care over an 
+extended period of time. Effect lasts for 10 turns and heals 10 health every 
+turn. DC 60 to hack. If nanobots are hacked, deals damage instead. PEGs are 
+easily detectable by tech scanning technologies or spells. PEGs cannot heal a 
+patient over their maximum health.
+
+==Improved P.E.G.s ==
+Adrenaline Reward:	50 Nanites
```

Comment:
>Adrenaline Reward:	50 Nanites

This isn't technically an Adrenaline reward, is it?

Also, are we strictly keeping Nanites out of doctor "spells" still? (Confirming in order to help me avoid false positives in my debug.)

Reply:
Wow. I can't believe I missed this. Thank you for catching it!